mobilecoinfoundation/attestation/tonic/build.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2023 The MobileCoin Foundation
```

Comment:
This file contains logic that's the same as the preceding build script (the difference appears to be the `protos` file name and some cmd arguments). Might you consider extracting this shared logic into a util function?

Reply:
That's a good question.
My initial thought was that only looking at these 2 crates and the differences it feels a bit heavy handed to make another crate just to house the common logic of invoking the command. The intent is to eventually publish these crates so wanted to minimize unnecessary churn there.

However this is also meant to be the start of moving most of the protobufs here to this repo.

I was thinking we need a plugin for grpcio, see #9, 
but perhaps there should be a follow on where there is a common build utilities crate that has `build_tonic_service`, `build_message_types`, and `build_grpcio_service` then if/when we are able to update the grpcio logic in #9 the consumers of `build_grpcio_service` will not need to update.

I've created #10 to track this further improvement